msl/examples/equipment/isotech/millik.py

```diff
@@ -28,9 +32,16 @@
 print('Connected devices:', thermometer.connected_devices)
 print('Available channel numbers:', thermometer.channel_numbers)
 
-# Read the current resistance values using a resistance value for thermistors
-print('Current resistance value for Channel 1:', thermometer.resistance(1, resis=20000, current='norm', wire=4))
-print('Current resistance values for all channels:', thermometer.read_all_channels(resis=20000, current='norm', wire=4))
+# Read the current resistance values using approximate resistance values for each channel
+r = [100, 470, 220, 820, 3300, 100, 1800, 13000]
+for i in range(8):  # here the sensors are all on the first millisKanner only
```

Comment:
Typically, one would use the following syntax when iterating through a list if you also want the corresponding item index
```python
for index, item in enumerate(my_list):
```

Reply:
and for your particular use case, you would include the `start` kwarg (which defaults to 0 if not specified)
```python
for channel, rng in enumerate(r, start=10)
```

Reply:
Oh that's much tidier. Thank you :)